packages/eslint-plugin-pf-codemods/lib/rules/datatoolbar-rename-toolbar.js

```diff
@@ -0,0 +1,58 @@
+const { getPackageImports } = require('../helpers');
+
+// https://github.com/patternfly/patternfly-react/pull/YOURNUMBERHERE
+module.exports = {
+  create: function(context) {
+    const imports = getPackageImports(context, '@patternfly/react-core')
+      .filter(specifier => specifier.imported.name === 'DataToolbar');
```

Comment:
great start, thanks for picking this up @evwilkin , but there are many more DataToolbar components :) Here are some more, not sure if I am missing any
```
DataToolbarContent,
  DataToolbarFilter,
  DataToolbarChip,
  DataToolbarItem,
```

Reply:
Thanks for clarifying - broadened the rule to rename any components containing 'DataToolbar' rather than exactly matching 'DataToolbar'

Reply:
`['DataToolbar', 'DataToolbarChipGroupContent', 'DataToolbarContent', 'DataToolbarExpandableContent', 'DataToolbarFilter', 'DataToolbarGroup', 'DataToolbarItem', 'DataToolbarToggleGroup'].includes(specifier.imported.name)`